docs/modules/ROOT/pages/how-tos/install/gcp.adoc

```diff
@@ -0,0 +1,251 @@
+    --role="${role}"
+done
+----
+<1> The project name from the inputs.
+<2> The cluster name from the inputs. The service account email must match the one previously created.
++
+[NOTE]
+====
+The documentation suggests to give the Service Account the `roles/owner` role.
+It also gives advice which roles to use if one does want to work with less privileges.
+The given list might not be complete or is ambiguous (name vs description).
+The first attempt with the above role set failed.
+The worker nodes did not come up.
+====
+
+6. Create Service Account Key [[service-account-key]]
++
+[source,bash]
+----
+gcloud iam service-accounts keys create key.json \
+  --iam-account openshift4-installer-<cluster name>@<project name>.iam.gserviceaccount.com <1>
+----
+<1> The cluster name from the inputs. The service account email must match the one previously created.
+
+7. Ensure firewall settings
++
+By default, no firewall is configured within a project.
+When using a project created by a third party, we need to check if the firewall settings meet the requirements from https://docs.openshift.com/container-platform/4.4/installing/install_config/configuring-firewall.html#configuring-firewall.
+
+== Create installer configuration
+
+Prerequisites:
+* GCP project setup according to above documentation
+* Service Account Key (key.json)
+
+.Required input
+`<base domain>`:: The base domain used to make the cluster available.
+`<cluster name>`::
+The name of the cluster.
+This name will be used as or as part of identifiers.
+It also will become a subdomain to the base domain.
+`<service account key>`:: See <<service-account-key>> 
+`<region>`:: The GCP region to place the cluster. Default to `europe-west6`
+
+1. Obtain the installer and pull secret
+
+   * https://cloud.redhat.com/openshift/install/gcp/installer-provisioned
+   * https://cloud.redhat.com/openshift/install/pull-secret
+
+2. Copy `<service account key>` to `~/.gcp/osServiceAccount.json`
+
+3. Crate an SSH key for that cluster
++
+[source,bash]
+----
+ssh-keygen -t rsa -b 4096 -N '' -f ~/.ssh/<cluster name> -C <cluster name>
+----
++
+NOTE: The key must be within the home directory to be picked up by the installer.
+
+4. Create the default installer config
++
+[source,bash]
+----
+openshift-install create install-config --dir=<cluster name>-config <1>
+----
+<1> The cluster name from the inputs.
++
+--
+.Answers
+SSH Public Key:: `~/.ssh/<cluster namme>`
+Platform:: gcp
+Project ID:: Will be extracted from the service account key file.
+Region:: <region>
+Base Domain:: <base domain>
+Cluster Name:: <cluster name>
+Pull Secret:: Grab from https://cloud.redhat.com/openshift/install/pull-secret.
+--
++
+.Credentials for Pull Secret
+****
+If the customer brings his own licences, the customer should obtain the Pull Secret using his RedHat account and produce us the Pull Secret.
+For all the other cases, use https://password.vshn.net/cred/detail/2420/.
```

Comment:
Should we have references to specific password manager entries in public docs?

Maybe rephrase to
```suggestion
For all the other cases, use the credentials in the password manager.
```

Reply:
Discussed that with Christian. Direct links to password.vshn.net would be possible, cause it does not reveal much data, but it is not that "beautiful". 
I would also rephrase it like your suggestion. Perhaps add the title of the password entry, something like:
"For all the other cases, use the credentials in the password manager at 'Red Hat RedHat CCSP Support Account'"

Reply:
Using the name as a reference can break badly. The name can change any time. Discussing this with @srueg, we agreed to keep the direct URL. The thinking goes like this:

* How tos within this repository are targeted for the use by VSHN and Puzzle.
* Where procedures differ between those two companies, it is indicated.

We will keep the direct URL but will state that this is for VSHN. Puzzle can then add their place when needed.